modules/UI/UI.js

```diff
@@ -40,6 +35,13 @@ import FollowMe from "../FollowMe";
 var eventEmitter = new EventEmitter();
 UI.eventEmitter = eventEmitter;
 
+/**
+ * Whether an overlay is visible or not.
+ * @type {boolean}
+ * @public
```

Comment:
does this really need to be public? looks like the `isOverlayVisible` getter should be used for checking and could be renamed to `_overlayVisible` and made private.